src/annotator/config/test/index-test.js

```diff
@@ -1,25 +1,22 @@
 'use strict';
 
 var proxyquire = require('proxyquire');
+var util = require('../../../shared/test/util');
 
 var fakeSharedSettings = {
   jsonConfigsFrom: sinon.stub(),
 };
 var fakeSettings = {};
 
-var configFrom = proxyquire('../index', {
+var configFrom = proxyquire('../index', util.noCallThru({
   './settings': fakeSettings,
   '../../shared/settings': fakeSharedSettings,
-});
+}));
```

Comment:
Lack of `noCallThru` here cost me a bunch of time when I had a typo in a test fake